armi/reactor/blocks.py

```diff
@@ -1582,12 +1582,18 @@ def getBoronMassEnrich(self):
         return b10 / total
 
     def getUraniumMassEnrich(self):
-        """Returns U-235 mass fraction assuming U-235 and U-238 only."""
-        u5 = self.getMass("U235")
-        if u5 < 1e-10:
+        """Returns fissile mass fraction of uranium."""
+        U = elements.bySymbol["U"]
+        compNucs = set(self.getNuclides())
+        uraniumNucs = set(nuc.name for nuc in U.nuclides)
+        uraniumNucsPresent = compNucs & uraniumNucs
```

Comment:
I see what you're doing here, and it seems correct.

But couldn't we also just do:

```python
self.getMass("U235") / self.getMass("U")
```

Aside from some error handling for zeros, obviously. But does `block.getMass("U")` do something subtlety different that I'm not thinking of?

Reply:
`self.getMass("U")` works for the denominator.

For the numerator, we want to add both fissile nuclides, `U233` and `U235`. There isn't a convenient method that does `getFissileNuclides()`, but since this method is specific to uranium we could just hard code it.